jwst/regtest/test_miri_mrs_spec2.py

```diff
@@ -58,11 +61,14 @@ def run_spec2_with_residual_fringe(rtdata_module):
             '--steps.residual_fringe.save_results=true',
             ]
 
-    Step.from_cmdline(args)
+    with resource_tracker.track():
+        Step.from_cmdline(args)
+
+
+def test_log_tracked_resources_spec2(log_tracked_resources, run_spec2_with_residual_fringe):
+    log_tracked_resources()
 
 
-@pytest.mark.slow
```

Comment:
side-question, is the `slow` mark still important/necessary here?

Reply:
It looks like this test is probably fast enough to require the tag any longer, though I'm not sure the cutoff was ever formalized. I can remove it.